### PR TITLE
feat: Modify SN to support A/B Test

### PR DESCRIPTION
**Pull request by zhounewz**
[https://app.asana.com/0/1202386226498661/1203253409172018](https://app.asana.com/0/1202386226498661/1203253409172018)

- [ ] Modify SN to support A/B Test

### CODE REVIEW

**Review by gedaaaa — APPROVED** [review]
LGTM
Please let zhen do some cleanup and release

File: packages/libs/lambda-at-edge/src/lib/pathToRegexStr.ts [hunk 1]
```diff
@@ -147,3 +151,81 @@ export const checkAndRewriteUrl = (
 
   debug(`[checkAndRewriteUrl] After: ${request.uri}, ${request.querystring}`);
 };
+
```
Comment (weiwensangsang) [inline packages/libs/lambda-at-edge/src/lib/pathToRegexStr.ts:154]: comment

File: packages/libs/lambda-at-edge/package.json [hunk 2]
```diff
@@ -83,7 +83,8 @@
     "node-fetch": "^2.6.1",
     "path-to-regexp": "^6.1.0",
     "send": "^0.17.1",
-    "zen-observable": "0.8.15"
+    "zen-observable": "0.8.15",
+    "murmurhash": "^2.0.1"
```
Comment (weiwensangsang) [inline packages/libs/lambda-at-edge/package.json:87]: You add a new package, need to test in stage, add new package is pretty dangerous for sn.

**Review by weiwensangsang — APPROVED** [review]

File: packages/libs/lambda-at-edge/package.json (hunk 2 above)
Comment (zhounewz) [reply packages/libs/lambda-at-edge/package.json:87]: ok

File: packages/libs/lambda-at-edge/src/lib/pathToRegexStr.ts (hunk 1 above)
Comment (zhounewz) [reply packages/libs/lambda-at-edge/src/lib/pathToRegexStr.ts:154]: added.

**Pull request merged**